### PR TITLE
Support for IBM DB2

### PR DESCRIPTION
**Pull request by shashiranjan84**
## Related Issues

- links https://transcend.height.app/inbox/T-15532

## Public **Changelog**

- Add support for DB2 driver

### CODE REVIEW

**Review by giacaglia — DISMISSED** [review]

**Review by anotherminh — APPROVED** [review]

**Comment by shashiranjan84** [comment]
@anotherminh does this need to be release publicly?

**Review by michaelfarrell76 — COMMENTED** [review]
please update minor version / add changelog entries for new functionality. patch version is for bug fixes.

was the driver itself installed on the dockerfile in another pr? probably should be added in same deploy

File: src/database.ts [hunk 1]
```diff
@@ -15,6 +15,7 @@ export const DatabaseIntegration = makeEnum({
   amazondynamodb: 'amazondynamodb',
   amazons3: 'amazons3',
   databrickslakehouse: 'databrickslakehouse',
+  ibmdb2: 'ibmdb2',
```
Comment (dipack95) [inline src/database.ts:18]: why do we not use camel case here?

**Review by dipack95 — APPROVED** [review]

File: src/database.ts (hunk 1 above)
Comment (shashiranjan84) [reply src/database.ts:18]: Is it question for me? I just followed the pattern here

**Comment by shashiranjan84** [comment]
> please update minor version / add changelog entries for new functionality. patch version is for bug fixes.
> 
> was the driver itself installed on the dockerfile in another pr? probably should be added in same deploy

Yes work is going on adding the driver but that change depends on this new driver type and blocking progress on another [PR](https://github.com/transcend-io/main/pull/25931)

**Comment by anotherminh** [comment]
> 1. was the driver itself installed on the dockerfile in another pr? probably should be added in same deploy

This seems like a tricky dependency between the two PRs then. @shashiranjan84 it sounds like you'll want to wait to merge this until you're pretty sure/ready to merge the app code in the monorepo.

> 
@anotherminh does this need to be released publicly?

Once this is merged, CI will publish the new version.

**Comment by shashiranjan84** [comment]
@anotherminh @michaelfarrell76 can I merge this PR now?

**Review by anotherminh — APPROVED** [review]
Go for it!

**Pull request merged**